### PR TITLE
build: fix failing workflows and builds

### PR DESCRIPTION
**Pull request by Benjozork**
## Summary of Changes

* Update to `@flybywiresim/map` 0.5.2
* Fix `dev-env` version
* Use local data instead of API for locating current flight
* **Note:** `origin` and `destination` do not currently work

Discord username (if different from GitHub): someperson#4953

## Testing instructions

* Ensure the EFB map locates your aircraft correctly
* Ensure the EFB map follows your aircraft as it moves
* Ensure no performance degradation

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Review by ZigTag — APPROVED** [review]
LGTM

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
         run: ./scripts/dev-env/run.sh npm run lint
       - name: lint-rust.sh
         run: |
-          docker run --rm -v "$(pwd)":/external ghcr.io/flybywiresim/dev-env:latest scripts/lint-rust.sh
+          docker run --rm -v "$(pwd)":/external ghcr.io/flybywiresim/dev-env@sha256:0bf0844b914cbd45a8a0a866a71c7a07fa09308f48cb961708be479f0d63aa0f
```
Comment (nistei) [inline .github/workflows/pr.yml:23]:
```suggestion
          docker run --rm -v "$(pwd)":/external ghcr.io/flybywiresim/dev-env@sha256:0bf0844b914cbd45a8a0a866a71c7a07fa09308f48cb961708be479f0d63aa0f scripts/lint-rust.sh
```

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -29,7 +29,7 @@ jobs:
         uses: actions/checkout@v2
       - name: test-rust.sh
         run: |
-          docker run --rm -v "$(pwd)":/external ghcr.io/flybywiresim/dev-env:latest scripts/test-rust.sh
+          docker run --rm -v "$(pwd)":/external ghcr.io/flybywiresim/dev-env@sha256:0bf0844b914cbd45a8a0a866a71c7a07fa09308f48cb961708be479f0d63aa0f
```
Comment (nistei) [inline .github/workflows/pr.yml:32]:
```suggestion
          docker run --rm -v "$(pwd)":/external ghcr.io/flybywiresim/dev-env@sha256:0bf0844b914cbd45a8a0a866a71c7a07fa09308f48cb961708be479f0d63aa0f scripts/test-rust.sh
```

**Review by nistei — CHANGES_REQUESTED** [review]

**Review by nistei — APPROVED** [review]

**Comment by Awemeter** [comment]
Category: QA Tester
Name: Awemeter#7919 
Date of testing: 2/16/2021
Version of the sim: 1.13.16.0
PR Tested: #3561 
Tier of testing: 1
Changes to observe:

- Update to @flybywiresim/map 0.5.2

- Fix dev-env version

- Use local data instead of API for locating current flight

- Note: origin and destination do not currently work

Testing technique: Description of how you tested the Pull Request
Steps you've made to test the Pull Request: Tried out
1. Spawned C&D at EGCB 
2. Started aircraft and fly pad and made sure plane shows in the proper place.
3. Started to move and take off and map updated every 10 seconds as it should.
4. Made sure the rest of EFB was still working.

Flight notes (When done a full flight):

Media: Insert screenshots, recordings, or anything relevant to the review here.

Issues (If Any): None


Overall Rating: Good
Conclusions of the testing: All good no issues found.

**Pull request merged**